libs/platform/src/lib/components/footer/footer.component.html

```diff
@@ -2,10 +2,12 @@
     <div class="fdp-footer--logo">
         <ng-content *ngTemplateOutlet="logo"></ng-content>
     </div>
-    <div class="fdp-footer--link">
-        <ng-content *ngTemplateOutlet="content"></ng-content>
-    </div>
-    <div class="fdp-footer--copy-right">
-        <ng-content *ngTemplateOutlet="copyright"></ng-content>
+    <div class="fdp-footer--container">
```

Comment:
this structure does not follow BEM. The modifier should be used alongwith the element to _modify_ it. Here, if `fdp-footer` is the block or element and `fdp-footer--link` is the modifier, it should be used as `<div class="fdp-footer fdp-footer--link">` and not plainly as `<div class="fdp-footer--link">`.
Also, `fdp-footer--container` does not seem like it is modifying anything; it should probably be its own element.